src/mlpack/methods/ann/layer/layer_norm_impl.hpp

```diff
@@ -19,10 +19,12 @@
 namespace mlpack {
 namespace ann { /** Artificial Neural Network. */
 
-template<typename InputDataType, typename OutputDataType>
+
+template <typename InputDataType, typename OutputDataType>
 LayerNorm<InputDataType, OutputDataType>::LayerNorm() :
-    size(10),
-    eps(1e-8)
+    size(1),
```

Comment:
I think we can revert back to the default size of 10 here now.

Reply:
Don't think that's necessary, the value has to be tailored to the task anyway, and I guess using ``LayerNorm<> layer();`` instead of ``LayerNorm<> layer(10);`` might create the wrong impression.

Reply:
Yes I agree with Marcus but I think that Shikhar meant to add the 10 which is also the default value. And here is another point where you could do a better job. Never design an example or test case where the default values are present because that may cause confusion (for the users) and mistakes (for you). I have seen default values in many of your examples. You especially like the values 10 and 1 for some reason :) lucky numbers for one of you?

Reply:
@zoq Please feel free to remove the default values for `BatchNorm` and `LayerNorm` parameter.